doc/changelog/03-notations/19149-existing_level.rst

```diff
@@ -0,0 +1,7 @@
+- **Added:**
+  :n:`at existing level`, this :n:`@syntax_modifier`
+  helps :ref:`factorize notations <NotationFactorization>`
+  with common prefixes. This is used in the
+  :cmd:`Reserved Notation` command
```

Comment:
```suggestion
  :n:`"at existing level"` option to :n:`@syntax_modifier`
  to help :ref:`factorize notations <NotationFactorization>`
    with common prefixes. This is used in the
  :cmd:`Reserved Notation` and other commands
```

Reply:
Thanks for the suggestion, unfortunately outdated now that Guillaume suggestions greatly simplifies everything.